src/tests/functions/gif/test_gif.py

```diff
@@ -12,7 +14,9 @@
         {"params": ["confused"]},
     ],
 )
+@patch.dict(os.environ, {"TENOR_API_TOKEN": "{}"})
 def test_gif_with_param(body):
+    """Test known keyword"""
```

Comment:
Technically, the test name describe what we are testing
We can keed the docstring if you want, we might want to add this to all tests in the future if we keep it